### PR TITLE
qa/tasks/ceph: redirect stderr to log file

### PR DESCRIPTION
**Pull request by tchaikov**
crimson write log to stderr, let's redirect it to log file for a more
peaceful teuthology.log.

Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
https://pulpito.ceph.com/kchai-2020-07-30_16:00:58-crimson-rados-wip-crimson-scrub2-distro-basic-smithi/

**Comment by tchaikov** [comment]
jenkins test signed

**Comment by tchaikov** [comment]
jenkins test make check

File: qa/tasks/ceph.py [hunk 1]
```diff
@@ -902,23 +912,20 @@ def cluster(ctx, config):
...
             try:
-                remote.run(
-                    args=[
-                        'sudo',
+                args = ['sudo',
                         'MALLOC_CHECK_=3',
                         'adjust-ulimits',
-                        'ceph-coverage',
-                        coverage_dir,
+                        'ceph-coverage', coverage_dir,
```
Comment (rzarzynski) [inline qa/tasks/ceph.py:918]: :+1:

**Pull request merged**